ota/README.md

```diff
@@ -0,0 +1,69 @@
+# Gateway OTA updater.
+
+This folder contains the package which deals with OTA updates for the gateway. The OTA process includes downloading, extracting and installing : node, build assets, native dependencies and `node_modules`. OTA does not update any of the underlying system libraries.
```

Comment:
NOTE: Like the rest of the document this is the end goal... I like splitting this kind of work over multiple pull requests.